manim/renderer/cairo_renderer.py

```diff
@@ -160,7 +184,27 @@ def show_frame(self):
         self.update_frame(ignore_skipping=True)
         self.camera.get_image().show()
 
-    def save_static_frame_data(self, scene, static_mobjects):
+    def save_static_frame_data(
```

Comment:
Every mention of static in this function should also be removed
```suggestion
    def save_frame_data(
```

Reply:
Same answer as above